scripts/gceworker.sh

```diff
@@ -54,6 +55,22 @@ case ${1-} in
     shift
     gcloud compute ssh "${name}" "$@"
     ;;
+    mount)
+    shift
+    if [ $# != 2 ]; then
+      echo "usage: $0 <remote-path> <local-path>"
```

Comment:
The idea is that most folks could just `./gceworker.sh mount` and it would do the right thing, namely let you edit your cockroach files locally.